src/localization/translations/nl.json

```diff
@@ -93,5 +93,11 @@
     "contact_add_success_toast": "Contact succesvol aangemaakt",
     "contact_add_no_consent_toast": "Zonder bevestiging gaat u niet door met het uitwisselen van inloggegevens en moet u de knop Weigeren gebruiken.",
     "contact_add_cancel_title": "Weet u zeker dat u wilt weigeren?",
-    "contact_add_cancel_message": "Het toevoegen van een contactpersoon is verplicht wanneer u een nieuwe partij tegenkomt in een credential uitwisselingsproces. Houd er rekening mee dat het hele proces voor het uitwisselen van inloggegevens wordt afgebroken als u bevestigt."
+    "contact_add_cancel_message": "Het toevoegen van een contactpersoon is verplicht wanneer u een nieuwe partij tegenkomt in een credential uitwisselingsproces. Houd er rekening mee dat het hele proces voor het uitwisselen van inloggegevens wordt afgebroken als u bevestigt.",
+    "onboarding_welcome_header": "Welkom",
+    "onboarding_welcome_title": "Sphereon Wallet",
+    "onboarding_welcome_intro_body": "Hiermee kunt u accreditaties, inloggegevens, badges en identiteitskenmerken ontvangen en delen.\n\nDeze zijn digitaal ondertekend en geattesteerd door externe uitgevende organisaties.",
+    "next_action_confirm": "Volgende",
+    "onboarding_welcome_store_body": "Slaat credentials van derden in versleutelde vorm rechtstreeks op uw apparaat op. Alleen u heeft toegang tot uw gegevens.\n\nHet communiceert niet met Sphereon-servers, tenzij u zich bijvoorbeeld aanmeldt voor back-ups of bedrijfsfuncties.",
```

Comment:
bedrijfsfuncties -> of zakelijke functies

Reply:
changed

Reply:
changed